### DISCUSSION TITLE
FileTransfer with the internet-facing instance of the demo server

### DISCUSSION THREAD

**Discussion by leitmap** [digitalpetri/opc-ua-demo-server#8] [General]
Hello, i'm using the internet-facing instance for some tests with FileTransfer.
While opening the file on the demo server in write mode, an error on my client side has occurred.
And now the file is still open on the server and i can not close it.
Is there a possibility to restart the demo server?
The FileHandle is not valid any more.

Thanks a lot and best regards
Martin

**Comment by kevinherron** [comment]
Normally yes, I could restart it for you right away, but I'm away for work this week and don't have access to my computer with the SSH keys on it. I won't be able to do it until I'm back home on Saturday.

In the meantime, you could try running the demo server locally.

**Reply by leitmap** [reply]
Thank you for your prompt reply. Perfect, I will use the local one.

Thank you again and best regards
Martin

**Reply by kevinherron** [reply]
I restarted the public demo server this morning.

**Reply by leitmap** [reply]
Thank you very much!
Best regards 
Martin
